src/SFA.DAS.QnA.Application/Validators/MaxWordCountValidator.cs

```diff
@@ -15,7 +15,9 @@ public class MaxWordCountValidator : IValidator
 
             if (!string.IsNullOrEmpty(text))
             {
-                var wordCount = text.Split(new[] { " " }, StringSplitOptions.RemoveEmptyEntries).Length;
+                char[] splitters = new char[] { ' ', '\r', '\n' };
+
+                var wordCount = text.Split(splitters, StringSplitOptions.RemoveEmptyEntries).Length;
```

Comment:
A more elegant solution:

```suggestion
                var wordCount = text.Split(' ', StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries).Length;
```
This will also detect other whitespace characters such as tabs, which shouldn't count as words according to the word count calculator that Alan linked.

Reply:
[This ](https://skillsfundingagency.atlassian.net/browse/QF-998) is the ticket describing the bug, and in it one can see that Alan researched and found this solution.

Reply:
The suggestion works if you apply it correctly:

![image](https://user-images.githubusercontent.com/119664159/225275144-3799334a-b192-4464-9c5b-440d354c0b08.png)

Reply:
I copied and pasted your code. First time fail, second time pass. Getting Alan to test it locally now.

Reply:
If you look at the screenshot I posted, you might wonder if NUnit is having trouble detecting that those absurdly long test cases are part of the same test. You can add the 10-word test I suggested to confirm if that's true.

Those long test cases should be in separate text files, not inline, if they're necessary at all.

Reply:
Upon further discussion with Alan, we are set on the initial solution. The goal is to have the backend and frontend solutions wordcount to be consistent. With Alan's suggested solution, the word count is consistent throughout (at 492, with the '8 words remaining' shown to the user correct), however with the second suggested solution the word count comes to 460 where the frontend comes to 492 (so the '8 words remaining' shown to users will be incorrect.) For this reason, we will be sticking with the first solution